src/main/java/com/bobocode/hoverla/bring/annotation/Bean.java

```diff
@@ -0,0 +1,22 @@
+package com.bobocode.hoverla.bring.annotation;
+
+import com.bobocode.hoverla.bring.context.ApplicationContext;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * Annotation which is used to mark types as beans in order to be scanned by {@link ApplicationContext}
+ */
+@Target(ElementType.TYPE)
```

Comment:
How about add `ElementType.METHOD` here? For Java Configuration

Reply:
I think it will be a part some of the stories related to java configuration